51a.md

```diff
@@ -0,0 +1,18 @@
+NIP-51a
+======
+
+User Public Key List
+-------------------------
+
+`draft` `optional` `author:fiatjaf` `author:arcbtc` `author:monlovesmango` `author:eskema` 
+
+An event with kind `30000` is defined as a list event (see [NIP-51 - Lists](51.md)) for user public keys. The list of tags should follow the format of kind 3 events as defined in [NIP-02 - Contact List and Petnames](02.md).
+
+Commonly established list parameter names for this kind are:
+- `mute` - a list of tags that the user wants to mute
+- `follow` - a list of tags that the user wants to follow
+- `pin` - a list of tags that the user wants to pin (for display on profile)
+- `bookmark` - a list of tags that the user wants to bookmark
```

Comment:
I'm confused. 51a is about "Public Keys" yet here you talk about tags? Isn't "tags" what we use for public list while for private list we use "content"?

```suggestion
- `mute` - a list of public keys that the user wants to mute
- `follow` - a list of public keys that the user wants to follow
- `pin` - a list of public keys that the user wants to pin (for display on profile)
- `bookmark` - a list of public keys that the user wants to bookmark
```

And what are bookmarked or pinned public keys for?

Reply:
all lists are comprised of tags. for 51a the list of tags should follow the format of kind 3 events as defined in [NIP-02 - Contact List and Petnames](https://github.com/monlovesmango/nips/blob/lists/02.md)

'pin' could be used to indicate "these are users I think are worth checking out"
'bookmark' could be used to save profiles you want to check out later